CHANGELOG.md

```diff
@@ -8,6 +8,10 @@ The following is a curated list of changes in the Enact agate module, newest cha
 
 - `Noto Sans` font as the default font
 
+### Fixed
+
+- `agate/ArcSlider`, `agate/Checkbox`, and `agate/TemperatureControl` style to match latest design for Silicon skin
```

Comment:
Why `Checkbox` is here?
Also, you didn't change `TemperatureControl` I think..

Reply:
I removed Checkbox and focused TemperatureControl knob size and color are changed from the ArcSlider change.